pandas/tests/io/parser/test_common.py

```diff
@@ -64,13 +66,14 @@ def _set_noconvert_columns(self):
         "parse_dates": parse_dates,
         "delimiter": ",",
     }
+    parser.engine = "c"
     parser._engine = MyCParserWrapper(StringIO(data), **parser.options)
 
     result = parser.read()
     tm.assert_frame_equal(result, expected)
 
 
-def test_empty_decimal_marker(all_parsers):
+def test_empty_decimal_marker(all_parsers, pyarrow_xfail):
```

Comment:
Can `pyarrow_xfail` be used as a decorator like `@skip_pyarrow` instead of supplied as an argument?

Reply:
yes. Done